src/Directory.Build.targets

```diff
@@ -9,6 +9,9 @@
   -->
   <ItemGroup>
     <EmbeddedResource Update="**\*.resx" GenerateSource="true" />
+
+    <!-- Also pass the resx files as additional files to enable entry validation -->
+    <AdditionalFiles Include="**\*.resx" />
```

Comment:
The order seems to matter for the rendering inside VS but not for the behavior. Let me know if you prefer it to appear as `EmbeddedResource` in VS.

Reply:
It seems fine. I was thinking more of the approach here: https://github.com/dotnet/roslyn/blob/master/docs/analyzers/Using%20Additional%20Files.md#passing-a-group-of-files, but either way is okay. I am also wondering if this entry (regardless of the one we choose) should be directly appended to the below string which gets added to `Microsoft.CodeAnalysis.Analyzers.targets`, so all analyzer authors see the new rules light up automatically. https://github.com/dotnet/roslyn-analyzers/blob/be84346550158fb6933154ffaae6ae2a2e93c198/src/Tools/GenerateGlobalAnalyzerConfigs/Program.cs#L372-L380

Reply:
@Evangelink You can revert this change once https://github.com/dotnet/roslyn-analyzers/pull/3924 goes in and you move to a newer analyzer package.

Reply:
This can now be reverted.